pkgs/by-name/el/elm-land/package.nix

```diff
@@ -0,0 +1,76 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, elmPackages
+, versionCheckHook
+, writeShellScript
+, nix-update
+, elm2nix
+, nixfmt-rfc-style
+}:
```

Comment:
I also prefer this style of formatting but seems we should probably just accept that C style have won.

Reply:
I just use nixfmt because it's required now for CI to pass

Reply:
I think it's failing https://github.com/NixOS/nixpkgs/actions/runs/13347842065/job/37280534110?pr=382369 because of this formatting.

My understanding is that it likes:

```nix
{
  foo,
  bar,
  baz,
}:
```

style but I'm yet to try it myself.